### PR TITLE
chore(github): add PR templates for default and release PRs

### PR DESCRIPTION
**Pull request by SanyaKochhar**
* Default PR template similar to OSS PR template, auto populates when you open a PR
* Release PR template can be used by adding `?expand=1&template=release_pull_request_template.md` to the PR URL during creation or copying the raw template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md
Comment (ksubrmnn) [inline .github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md:21]: Is there any action item for this? Can you give more context as to why this question is being asked and answered?

Comment (SanyaKochhar) [reply .github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md]: nope, it was a relic from the upstream checklist. Removed now

**Comment by nshankar13** [comment]
For the release PRs, I would add the following notes: 

- I have verified that the images for the new versions of osm-controller, init, and osm-injector are in MCR. 
- This PR has received 3 approvals before merging.

**Review by ksubrmnn — DISMISSED** [review]

**Review by nshankar13 — APPROVED** [review]

**Review by ksubrmnn — APPROVED** [review]

**Pull request merged**
